packages/hasher/package.json

```diff
@@ -17,9 +17,8 @@
   },
   "dependencies": {
     "@yarnpkg/lockfile": "^1.1.0",
-    "backfill-config": "^2.0.6",
-    "backfill-logger": "^2.0.6",
-    "fast-glob": "^3.0.4",
```

Comment:
Why was fast-glob removed here? It's used here:
backfill/packages/hasher/src/hashOfFiles.ts
backfill/packages/hasher/src/yarnWorkspaces.ts